erpnext/accounts/party.py

```diff
@@ -613,3 +616,26 @@ def get_partywise_advanced_payment_amount(party_type, posting_date = None):
 
 	if data:
 		return frappe._dict(data)
+
+def get_default_billing_contact(doctype, name):
+	""" 
+		Returns default contact for the given doctype and name.
+		Can be ordered by `contact_type` to either is_primary_contact or is_billing_contact.
+	"""
+	out = frappe.db.sql("""
+			SELECT dl.parent, c.is_primary_contact, c.is_billing_contact
+			FROM `tabDynamic Link` dl
+			INNER JOIN tabContact c ON c.name = dl.parent
+			WHERE 
+				dl.link_doctype=%s AND
+				dl.link_name=%s AND
+				dl.parenttype = "Contact"
+			ORDER BY is_billing_contact DESC, is_primary_contact DESC
+		""", (doctype, name))
```

Comment:
@ronelvcabrera This should be party_type and not "Sales Invoice" which is in this case. Contact will linked to customers and not Sales Invoices. Please update